api/api_full.go

```diff
@@ -147,6 +147,9 @@ type FullNode interface {
 	StateCompute(context.Context, abi.ChainEpoch, []*types.Message, types.TipSetKey) (*ComputeStateOutput, error)
 
 	MsigGetAvailableBalance(context.Context, address.Address, types.TipSetKey) (types.BigInt, error)
+	MsigCreate(context.Context, int64, []address.Address, types.BigInt, address.Address, types.BigInt) (AddressPair, error)
+	MsigPropose(context.Context, address.Address, address.Address, types.BigInt, address.Address, uint64, string) (MsigProposeOutput, error)
```

Comment:
Params should just be raw `[]byte`s, not hex encoded string

Reply:
Changed `params` from `string` to `[]byte` in both `MsigPropose` and `MsigApprove`